java/ql/test/library-tests/frameworks/android/taint-database/FlowSteps.java

```diff
@@ -27,155 +27,160 @@
 	}
 
 	public static String[] appendSelectionArgs() {
-		String[] originalValues = taint();
-		String[] newValues = taint();
+		String[] originalValues = {taint()}; // $hasTaintFlowStep
+		String[] newValues = {taint()}; // $hasTaintFlowStep
 		return DatabaseUtils.appendSelectionArgs(originalValues, newValues);
 	}
 
 	public static String concatenateWhere() {
-		String a = taint();
-		String b = taint();
+		String a = taint(); // $hasTaintFlowStep
```

Comment:
I think this test becomes a bit easier to read if we rename the label to something like "taintReachesReturn" - that way you don't need to look into the flow configuration to figure out what's being tested.